### ISSUE TITLE
The slicer cannot hold knife and axe items on latest versions of Create Slice & Dice [2.3.3 -> 2.3.5]

### ISSUE THREAD

**Issue by kkempfer** [PssbleTrngle/SliceAndDice#159]
### What happened?

The slicer cannot hold knife and axe items in latest versions of your mod for Fabric. I did not test the behavior on Forge.

The tag is not assigned. See log output below.

I confirm having tested on a minimal setting:

Create: 0.5.1-c-build.1160+mc1.19.2
Fabric API: 0.76.1+1.19.2
Fabric Language Kotlin: 1.11.0+kotlin.2.0.0
Farmer's Delight: 1.19.2-1.3.10.2
Create Slice & Dice: 2.3.3+

The slicer accepts knives and axes in version 2.3.2.

### Loader

fabric

### Mod Version

2.3.5

### Minecraft Version

1.19

### Relevant log output

```Shell
[11:11:53] [Worker-ResourceReload-9/ERROR]: Couldn't load tag sliceanddice:allowed_tools as it is missing following references: #minecraft:axes (from Create Slice & Dice)
```
```

**Comment by PssbleTrngle** [comment]
Weird, could you sent me your entire log file?

**Comment by kkempfer** [comment]
Here are the logs using version 2.3.2 (working) and 2.3.3 (not working). Same error for later versions up to the latest 2.3.5.

[latest-2.3.2.log](https://github.com/user-attachments/files/16366327/latest-2.3.2.log)
[latest-2.3.3.log](https://github.com/user-attachments/files/16366328/latest-2.3.3.log)

**Comment by kkempfer** [comment]
FYI, I used Create 0.5.1.c and Fabric API 0.76.1 for my tests. If I update them to their latest versions, i.e. Create 0.5.1.f Patch 1 and Fabric API 0.77.0, I still get the same error with Create Slice & Dice.

Corresponding log: [latest-create-0.5.1f-sliceanddice-2.3.5.log](https://github.com/user-attachments/files/16366394/latest-create-0.5.1f-sliceanddice-2.3.5.log)

**Comment by PssbleTrngle** [comment]
I was not aware that the `#minecraft:axes` tag was only added in 1.20, should be fixed in the next version for 1.19 too
